files/scripts/configdb-load.sh

```diff
@@ -7,7 +7,7 @@ done
 
 # If there is a config db dump file, load it
 if [ -r /etc/sonic/config_db.json ]; then
-    sonic-cfggen -j /etc/sonic/config_db.json --write-to-db
+    sonic-cfggen -j /etc/sonic/init_cfg.json -j /etc/sonic/config_db.json --write-to-db
```

Comment:
What if /etc/sonic/init_cfg.json doesn't exist? This command will fail. Need another check for /etc/sonic/init_cfg.json.

Reply:
will add if else condition to decide whether /etc/sonic/inint_cfg.json exist or not.

Reply:
I don't think it's necessary to load init_cfg.json alone here if there is no config_db dump file. I think the following is sufficient:

```
        # If there is a config db dump file, load it
        if [ -r /etc/sonic/config_db.json ]; then
            if [ -r /etc/sonic/init_cfg.json ]; then
                sonic-cfggen -j /etc/sonic/init_cfg.json -j /etc/sonic/config_db.json --write-to-db
            else
                sonic-cfggen -j /etc/sonic/config_db.json --write-to-db
            fi
        fi
```

Reply:
Fixed.